inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/StreamSinkController.java

```diff
@@ -85,4 +87,20 @@ public Response<Boolean> delete(@PathVariable Integer id) {
         return Response.success(result);
     }
 
+    @DeleteMapping(value = "/sink/deleteByRelatedId/{groupId}/{streamId}/{sinkName}")
+    @OperationLog(operation = OperationType.DELETE)
+    @ApiOperation(value = "Delete stream sink by groupId, streamId and sinkName")
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = "groupId", dataTypeClass = String.class, readOnly = true),
```

Comment:
Why add `readOnly` for the API document here?

Reply:
it should be **_required_**, just mistake